Makefile

```diff
@@ -66,8 +66,11 @@ AWS_SEC_ACC_KEY := $(shell aws configure get aws_secret_access_key)
 BUCKET=$(USER)-tablesnap-tests
 HARDLINKS_PATH=/tmp/tablesnap-hardlinks
 
-docker-test-tablesnap: docker-build
-	docker run -it migrate2to3:v1 ./test_tablesnap.sh $(DIRECTORY) $(TIMEOUT_A) $(TIMEOUT_B) $(AWS_ACC_KEY_ID) $(AWS_SEC_ACC_KEY) $(BUCKET) $(HARDLINKS_PATH)
+test-tablesnap-upload-py2.7: docker-build
+	docker run -it migrate2to3:v1 ./test_tablesnap.sh python2 $(DIRECTORY) $(TIMEOUT_A) $(TIMEOUT_B) $(AWS_ACC_KEY_ID) $(AWS_SEC_ACC_KEY) $(BUCKET) $(HARDLINKS_PATH)
+
+test-tablesnap-upload-py3.6: docker-build
+	docker run -it migrate2to3:v1 ./test_tablesnap.sh python3 $(PYTHON2) $(DIRECTORY) $(TIMEOUT_A) $(TIMEOUT_B) $(AWS_ACC_KEY_ID) $(AWS_SEC_ACC_KEY) $(BUCKET) $(HARDLINKS_PATH)
```

Comment:
Oh, I think you need to remove the `$(PYTHON2)` argument right after the `python3`. It turns out that doesn't hurt anything because it expands to an empty string, but it's confusing to read.

Reply:
Good catch. It's true, when I ran the test, there was no error. Removed $(PYTHON2) & re-tested rule.